static/main.css

```diff
@@ -224,3 +220,136 @@ textarea:focus {
     opacity: 0;
   }
 }
+.Footer {
+  background-color: #c49542;
+  font-family: Roboto;
+  padding-bottom: 3rem;
+  padding-top: 3rem;
+  text-align: center;
+}
```

Comment:
It might be fun to explore `primaryFont`, `secondaryFont` stylus variables

As well as variables for standard paddings:

`unit`, `double`, `triple` paddings